cosmos/converter.py

```diff
@@ -294,6 +297,9 @@ def __init__(
         self.dbt_graph.load(method=render_config.load_method, execution_mode=execution_config.execution_mode)
 
         self._add_dbt_project_hash_to_dag_docs(dag)
+        self._store_cosmos_telemetry_metadata_on_dag(
+            dag, render_config, project_config, operator_args, initial_load_method
```

Comment:
We should not need `operator_args` for the DAG render config 
```suggestion
            dag, render_config, project_config, initial_load_method
```